react/styles.css

```
@@ -1 +1,3 @@
-.container {}
+:global(.vtex__icon-spinner) {
+    position: aboslute;
```

This can't be working 😆 
```suggestion
    position: absolute;
```

---

I will improve the css. It's not last solution for this project.

---

Where can I add a comment ?

---

I wanna render my app only when the Stylesheets was created inside of card-form-ui. 
I'm currently using `<Spinner />` but I don't know if it's the best way to solve this UI problem. But maybe this change should not be part of this PR.

---

You confused me with your comments lol. This is fine now that you fixed it 😆